doc/ci_admin_guide/Docker_registry_provisioning.md

```diff
@@ -0,0 +1,152 @@
+```
+
+<details><summary>Example list of images (updated 2018-06-22)</summary>
+```
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-glance-api:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-glance-registry:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-compute:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-placement-api:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-novncproxy:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-api:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-conductor:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-consoleauth:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-ssh:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-scheduler:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-keystone:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-neutron-server:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-neutron-metadata-agent:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-account:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-object:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-rsyncd:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-object-expirer:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-proxy-server:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-container:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-barbican-api:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-barbican-keystone-listener:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-barbican-worker:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-heat-api:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-heat-api-cfn:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-heat-engine:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-horizon:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-swift-base:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-memcached:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-cron:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-rabbitmq:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-kolla-toolbox:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-nova-libvirt:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-fluentd:ocata
+ci-repo.englab.juniper.net:5000/kolla/centos-binary-mariadb:ocata
+ci-repo.englab.juniper.net:5000/contrail-openstack-neutron-init:ocata-master-91
+ci-repo.englab.juniper.net:5000/contrail-openstack-compute-init:ocata-master-91
+```
+</details>
+
+Then, copy this file to the registry, and pull the images:
+
+```bash
+for image in `cat pullme`; do docker pull $image; done
+```
+
+### Starting the registry
+
+> NOTE: This section is adapted from [Docker registry deploying instruction][docker-registry-deploying].
```

Comment:
Very nice to see reference to original documentation.

BTW, maybe we should consider writing something about volumes here as well (as discussed on chat?)